data/com/mojang/math/Vector4f.mapping

```diff
@@ -1,6 +1,25 @@
 CLASS com/mojang/math/Vector4f
+	METHOD <init> (FFFF)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 w
+	METHOD <init> (Lcom/mojang/math/Vector3f;)V
+		ARG 1 other
+	METHOD add (FFFF)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 w
 	METHOD dot (Lcom/mojang/math/Vector4f;)F
-		ARG 1 vector
+		ARG 1 other
+	METHOD equals (Ljava/lang/Object;)Z
+		ARG 1 other
+	METHOD lerp (Lcom/mojang/math/Vector4f;F)V
+		ARG 1 other
+		ARG 2 percentage
+	METHOD mul (F)V
+		ARG 1 multiplier
```

Comment:
I've seen `multiplier` and `scale`, needs standarization.

Reply:
Looked at the other vector and matrix classes, and changed parameters so if the method name is `scale`, the parameter is `scale`, and if the method name is `mul`/`multiply`, then `multiplier`.